src/core/styles/variables.css

```
@@ -0,0 +1 @@
+body {}
```

Stylelint (17.9.0)

[error] Empty block (block-no-empty)

(block-no-empty)